.github/workflows/main.yml

```diff
@@ -87,7 +87,7 @@ jobs:
           - { dep: "nltk gpt2", extra: "termcolor>=1.1.0", testfile: examples/gpt2_test.py }
           - { dep: elastic, testfile: tests/wrappers/elastic_indexers_test.py }
           - { dep: faiss, testfile: tests/wrappers/faiss_indexers_test.py }
-          - { dep: "huggingface nltk", testfile: tests/wrappers/huggingface }
+          - { dep: "huggingface nltk", extra: "tensorflow>=2.5.0", testfile: tests/wrappers/huggingface }
```

Comment:
Is `nltk` still needed here?

Reply:
Yes, I think one of Jenny's unittest need it

Reply:
i see, missed it on that PR. Will approve this.